src/components/eventsSection/ActionContainer.tsx

```diff
@@ -36,10 +43,12 @@ export default function ActionContainer({ section }: ActionSectionProps) {
               {section?.ctaSubsectionDescription}
             </h3>
             <div className="flex sm:float-left">
-              <SecondaryButton
-                buttonText={section?.ctaButton?.buttonText}
-                btnClasses={`bg-white text-secondaryBlack px-[60px]`}
-              />
+              <Link href={`mailto:${email}`}>
```

Comment:
In this case, it should be a link and not a button inside Link.
You just can style link as a button